Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -257,6 +264,43 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
                                resourcesBuildPhase: resourcesBuildPhase)
     }
 
+    func generateCopyFilesBuildPhases(path: AbsolutePath,
```

Comment:
I'd add a unit test that ensures that the project contains the right copy files build phases.